RELEASE-NOTES.txt

```diff
@@ -10,6 +10,7 @@
 - [*] Payments: Add account type field to receipts [https://github.com/woocommerce/woocommerce-ios/pull/9416]
 - [*] Products can now be filtered within Order creation [https://github.com/woocommerce/woocommerce-ios/pull/9258]
 - [*] Products: Adds read-only support for the Composite Products extension in the Products list, including a list of components in product details. [https://github.com/woocommerce/woocommerce-ios/pull/9455]
+- [Internal] Store onboarding: Mark "Launch your store" task as complete if the store is already public. [https://github.com/woocommerce/woocommerce-ios/pull/9481]
```

Comment:
nit: might be worth mentioning briefly that this is a workaround for a backend issue, so it's not a feature